formats/calendar/resources/ext.srf.formats.eventcalendar.js

```diff
@@ -467,6 +471,30 @@
 							// If the day number (where available) is clicked then switch to the daily view
 							if ( allDay && data.query.ask.parameters.dayview && $( jsEvent.target ).is( 'div.fc-day-number' ) ) {
 								container.fullCalendar( 'changeView', 'agendaDay'/* or 'basicDay' */).fullCalendar( 'gotoDate', date );
+							} else {
```

Comment:
@kghbln  , @mwjames 
 I have built a separate function and now wait for a Review.
My Regards